src/TBuffer.cpp

```diff
@@ -603,10 +604,29 @@ inline int TBuffer::lookupColor(const QString & s, int pos )
     return -1; // unbeendete sequenz
 }
 
-
-void TBuffer::translateToPlainText( std::string & s )
+void TBuffer::translateToPlainText( std::string & s, const bool isFromServer )
 {
-    //cout << "TRANSLATE<"<<s<<">"<<endl;
+    std::string localBuffer; // As well as enabling the prepending of left-over
```

Comment:
Why not a comment above the text - it'll take up less lines and be more clang-format friendly.